docs/community/100-days-of-oss.md

```diff
@@ -0,0 +1,71 @@
+
+## The Official Kickoff
+We're starting 100 days from the end of [Hacktoberfest](https://hacktoberfest.com/), a month-long celebration of open source contributions. Starting July 23rd, we'll provide continuous support, daily inspirational tweets, and engaging events to help you stay motivated and make progress. 
+
+Don't worry if you're unable to start on the same day as everyone else. The #100DaysOfOSS challenge is flexible, and you can join in whenever you're ready. Just jump in at any point and begin with day one of your personal challenge. 
+
+## Support
+To make the most of your #100DaysOfOSS journey, here are some additional resources and events you can explore:
+
+1. **Weekly Twitter Spaces:** Join our weekly Twitter Spaces sessions where we discuss open source topics, share insights, and connect with like-minded individuals. Follow us on [Twitter](https://twitter.com/saucedopen) to stay updated on upcoming sessions.
+
+2. **Community Events:** Discover a wide range of events on our community docs page. Whether it's hack days, workshops, or office hours, these events provide excellent opportunities to learn, collaborate, and find new projects to contribute to. Visit our [community docs page](https://docs.opensauced.pizza/community/welcome-to-the-community) to explore the upcoming events.
+
+3. **Weekly Contribution Opportunities:** If you're actively looking for open source projects to contribute to, check out the [weekly post](https://dev.to/bekahhw/series/23323) for new contribution opportunities.
+
+4. **Weekly Office Hours:** Have questions or need help? Join us on [Discord](https://discord.gg/dXvGd8mz) during our office hours or post in our #100DaysOfOSS channel. We're here to help you succeed!
+
+## Where to Start?
+If you're ready to start your #100DaysOfOSS journey, here are some tips to help you get started:
+- **Find a Project:** Explore the [OpenSauced](https://opensauced.pizza/) website to find a project that interests you. You can also check out the [weekly post](https://dev.to/bekahhw/series/) for exciting contribution opportunities.
+- **Take our Intro To Open Source Course:** If you're new to open source, we recommend taking our [Intro to Open Source](https://github.com/open-sauced/intro) course to learn more about open source and how to get started.
+
+## The Power of the #100DaysOfOSS Challenge
+The #100DaysOfOSS challenge offers a supportive community where developers can find encouragement, share experiences, and overcome roadblocks together. 
+
+Our hope is that the community will provide a safe space to discuss challenges, celebrate achievements, and exchange insights, creating an environment that helps individuals stay on track and avoid giving up.
+
+
+## Why join the #100DaysOfOSS Challenge:
+
```

Comment:
[Nitpick] I'd mention something like tuition-fee free. It’ll pique new contributor’s interest.